config/environments/production.rb

```diff
@@ -18,7 +18,7 @@
 
   # Ensures that a master key has been made available in either ENV["RAILS_MASTER_KEY"]
   # or in config/master.key. This key is used to decrypt credentials (and other encrypted files).
-  # config.require_master_key = true
+  config.require_master_key = false
```

Comment:
What's the implication for this change? Does Docker not work with this turned on?

Reply:
This was left over from debugging. Everything works fine now when keeping this setting to true, so I removed the change.